templates/quarkus/manifest.yaml

```diff
@@ -1,10 +1,8 @@
+builders:
+  default: quay.io/boson/faas-jvm-builder:latest
+  jvm: quay.io/boson/faas-jvm-builder:latest
+  native: quay.io/boson/faas-quarkus-native-builder:latest
```

Comment:
These lines instruct the system to _always_ use these builders, regardless of Buildpack vs S2I.  Therefore we should probably not include them and rely on the in-code defaults.

This is what we are discussing now; how best to specify defaults for S2I and Buildpacks without them interfering, but still easy to patch in places like this.